shared/javacpp/setupBuild.gradle

```diff
@@ -5,6 +5,26 @@ apply plugin: 'edu.wpi.first.NativeUtils'
 apply from: "${rootDir}/shared/config.gradle"
 apply from: "${rootDir}/shared/javacommon.gradle"
 
+wpilibTools.deps.wpilibVersion = wpi.versions.wpilibVersion.get()
+
+def nativeConfigName = 'wpilibNatives'
```

Comment:
It's ok if multiple projects share this name right?

Reply:
It builds lol

Reply:
And it runs in test mode and the shadow jar and robot sim and everything? This could cause unsatisfied link errors if you aren't careful

Reply:
Test mode worked, I didn't run the simulation